R/match_on_ltcs.R

```diff
@@ -0,0 +1,100 @@
+#' Match on LTC changed_dob and dates of LTC incidence (based on hospital incidence only)
+#'
+#' @description Match on LTC changed_dob and dates of LTC incidence (based on hospital incidence only)
+#'
+#' @param
+#'
+#' @return
+#' @export
+#'
+#' @examples
+match_on_ltcs <- function(data, year) {
+  # Match on LTC lookup
+  matched <- dplyr::left_join(
+    data,
+    readr::read_rds(get_ltcs_path(year)),
+    by = "chi",
+    suffix = c("", "_ltc")
+  ) %>%
+    # Replace any NA values with 0 for the LTC flags
+    dplyr::mutate(dplyr::across("arth":"digestive", ~ tidyr::replace_na(., 0)))
+  matched <- correct_demographics(matched, year)
+  return(matched)
+}
+
+correct_demographics <- function(data, year) {
+  # Checking and changing DOB and age
+  # data <- data %>%
+  test <- data %>%
```

Comment:
```suggestion
  test <- data %>%
```

Also rename the object from `test` to something meaningful.